include/onnxruntime/core/framework/stream_handles.h

```diff
@@ -164,4 +166,25 @@ class IStreamCommandHandleRegistry {
   virtual void RegisterCreateStreamFn(OrtDevice::DeviceType device_type, CreateStreamFn f) = 0;
 };
 
+struct CpuBuffersInfo {
```

Comment:
the CpuBuffersInfo is special baked for cuda EP, don't put it into the core/framework folder.